### ISSUE TITLE
checksum of pre built binaries didn't match at v1.14.1

### ISSUE THREAD

**Issue by suzuki-shunsuke** [mesosphere/mindthegap#744]
https://github.com/mesosphere/mindthegap/releases/tag/v1.14.1

Download all assets with GitHub CLI. https://cli.github.com/manual/gh_release_download

```sh
gh -R mesosphere/mindthegap release download v1.14.1
```

Checksum verification with sha256sum fails.

```console
$ sha256sum -c checksums.txt
mindthegap_v1.14.1_darwin_amd64.tar.gz: OK
mindthegap_v1.14.1_darwin_arm64.tar.gz: FAILED
mindthegap_v1.14.1_linux_amd64.tar.gz: OK
mindthegap_v1.14.1_linux_arm64.tar.gz: OK
mindthegap_v1.14.1_windows_amd64.zip: FAILED
mindthegap_v1.14.1_windows_arm64.zip: FAILED
sha256sum: WARNING: 3 computed checksums did NOT match
```

```console
$ sha256sum -c checksums.txt | grep FAILED
mindthegap_v1.14.1_darwin_arm64.tar.gz: FAILED
mindthegap_v1.14.1_windows_amd64.zip: FAILED
sha256sum: mindthegap_v1.14.1_windows_arm64.zip: FAILED
WARNING: 3 computed checksums did NOT match
```

checksums.txt

```
a508cc65d9ca5642ef3cbe1e8965e3a90306452d99953ee25b1873f4389f0e7e  mindthegap_v1.14.1_darwin_amd64.tar.gz
ef809b87ec2452433e60c634d02f2ab37c6a3097aae2e7edec44c5ed8ccd84ad  mindthegap_v1.14.1_darwin_arm64.tar.gz
e4d0fe1bfa4eaef93edfe621d2ab346696be085c9152be430f627c1da03f017b  mindthegap_v1.14.1_linux_amd64.tar.gz
1e4b9386223e64280ac4c2c68b6a337aeb911c7ec5a252fedb76d01972f6c765  mindthegap_v1.14.1_linux_arm64.tar.gz
4d445e1b687ff6cda26dddcb1f0a0ee5de0c2a98249ceb8c36a442c1020a0612  mindthegap_v1.14.1_windows_amd64.zip
3c999ccdbbfed653b0cd842910edb9fd411e0ae5fab0aabc3166113abae97368  mindthegap_v1.14.1_windows_arm64.zip
```

```console
$ sha256sum *
c60878b8e54fe76b70c5214870986cf64031a8f2934abbc7411cf73324bc2faa  checksums.txt
a508cc65d9ca5642ef3cbe1e8965e3a90306452d99953ee25b1873f4389f0e7e  mindthegap_v1.14.1_darwin_amd64.tar.gz
bed0dede9a10fddae46e3124d05fa39398a4e38bf64adaad15a51f6798862b74  mindthegap_v1.14.1_darwin_arm64.tar.gz
e4d0fe1bfa4eaef93edfe621d2ab346696be085c9152be430f627c1da03f017b  mindthegap_v1.14.1_linux_amd64.tar.gz
1e4b9386223e64280ac4c2c68b6a337aeb911c7ec5a252fedb76d01972f6c765  mindthegap_v1.14.1_linux_arm64.tar.gz
98e353d7051158262a35ceafd4298283b0aa9e57d68ba3cc3bd98d53eaa06fec  mindthegap_v1.14.1_windows_amd64.zip
75dea1e535966529d35b439d93242d740c3089cc751cc3ba3550df20acd0bc75  mindthegap_v1.14.1_windows_arm64.zip
```

**Comment by jimmidyson** [comment]
Thank you for the report! This was an error that triggered the release workflow multiple times concurrently. Building tarballs is non-deterministic right now as it embeds creation timestamps so checksums was uploaded from one job and some artifacts from another. Hopefully this is now fixed in v1.14.3 - coming very soon!

**Comment by jimmidyson** [comment]
Release succeeded (https://github.com/mesosphere/mindthegap/actions/runs/9469885882/job/26089574833) and tested that checksums are correct:

```
$ gh -R mesosphere/mindthegap release download v1.14.3

$ sha256sum -c checksums.txt
mindthegap_v1.14.3_darwin_amd64.tar.gz: OK
mindthegap_v1.14.3_darwin_arm64.tar.gz: OK
mindthegap_v1.14.3_linux_amd64.tar.gz: OK
mindthegap_v1.14.3_linux_arm64.tar.gz: OK
mindthegap_v1.14.3_windows_amd64.zip: OK
mindthegap_v1.14.3_windows_arm64.zip: OK
```
